<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartOfAccountWindow</class>
 <widget class="QMainWindow" name="ChartOfAccountWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>754</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Narrow</family>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ChartOfAccountWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

background-color:rbg 54,15,1;
color:black;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="Container">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>500</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:10px;
	background-color:grey;
	color:white;
}
QTableWidget {
    background-color: #2e2e2e; /* Dark background for the table */
    selection-background-color: #4caf50; /* Green for selected rows */
    selection-color: white; /* White text on selection */
    color: #e0e0e0; /* Light gray text for table content */
    border: 1px solid #555555; /* Dark gray border */
}

QHeaderView::section {
    background-color: #3b3b3b; /* Darker gray for header */
    padding: 4px;
    font-weight: bold;
    color: #ffffff; /* White header text */
    border: 1px solid #555555; /* Dark gray border for header */
}

QTreeWidget::item {
    padding: 5px;
}

QTreeWidget::item:has-children {
    background-color: #2b2b2b; /* Dark gray */
    font-weight: bold;
    color: #ffffff; /* White text */
    height: 40px; /* Increase row height */
}

QTreeWidget::item:selected:has-children {
    background-color: #444444; /* Slightly lighter dark gray */
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="UpperLogoBar">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:black;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QWidget" name="LogoStack">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <widget class="QTextEdit" name="Motto">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>20</y>
             <width>914</width>
             <height>80</height>
            </rect>
           </property>
           <property name="cursor" stdset="0">
            <cursorShape>IBeamCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-bottom:3px;
color:white;</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;ACCOUNTING MADE EASY&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>10</y>
             <width>300</width>
             <height>100</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>100</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
           <property name="icon">
            <iconset>
             <activeon>:/images/AppLogo.png</activeon>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>300</width>
             <height>100</height>
            </size>
           </property>
          </widget>
          <zorder>pushButton</zorder>
          <zorder>Motto</zorder>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="AccountWindow">
         <property name="font">
          <font>
           <family>Arial Narrow</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QWidget" name="AccountWindowStacked">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item alignment="Qt::AlignRight">
            <widget class="QFrame" name="Buttons">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>80</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="CreateAccount">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">padding: 5</string>
                </property>
                <property name="text">
                 <string>Create Account</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="BasicInfo">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial Narrow</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Here are your list of Accounts:</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="ChartOfAccountFrame">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>400</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item alignment="Qt::AlignHCenter">
               <widget class="QFrame" name="DeleteAccountFrame">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QLineEdit" name="DeleteLabel">
                   <property name="minimumSize">
                    <size>
                     <width>500</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>Arial Narrow</family>
                     <pointsize>15</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Selected Account</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_2">
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                     <widget class="QPushButton" name="Cancel">
                      <property name="minimumSize">
                       <size>
                        <width>80</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>80</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>CANCEL</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="EditAccount">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>130</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="cursor">
                       <cursorShape>PointingHandCursor</cursorShape>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">padding:5;
</string>
                      </property>
                      <property name="text">
                       <string>Edit Account</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="Delete">
                      <property name="minimumSize">
                       <size>
                        <width>80</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>80</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">background-color:grey;
color:white;</string>
                      </property>
                      <property name="text">
                       <string>DELETE</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QTreeWidget" name="ChartOfAccount">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Arial Narrow</family>
                  <pointsize>15</pointsize>
                 </font>
                </property>
                <property name="mouseTracking">
                 <bool>false</bool>
                </property>
                <property name="autoFillBackground">
                 <bool>true</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color:black;
color:white;
</string>
                </property>
                <property name="lineWidth">
                 <number>10</number>
                </property>
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="horizontalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="sizeAdjustPolicy">
                 <enum>QAbstractScrollArea::AdjustToContents</enum>
                </property>
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="alternatingRowColors">
                 <bool>false</bool>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::NoSelection</enum>
                </property>
                <attribute name="headerDefaultSectionSize">
                 <number>300</number>
                </attribute>
                <column>
                 <property name="text">
                  <string>Code</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Name</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>TYPE</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>BALANCE</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
